sdk/granter_test.go

```diff
@@ -552,3 +596,115 @@ func TestReasonerSpecificity(t *testing.T) {
 		})
 	}
 }
+
+func TestReasonerSpecificityWithNamespaces(t *testing.T) {
```

Comment:
Can we update the table test message to make it clear what is happening and what the expected output should be.

Reply:
@elizabethhealy I should have provided an example but I was thinking like `expect namespace kas when no value or definition grant`

Reply:
updated